### PR TITLE
chore: Platform get schedule

### PR DESCRIPTION
**Pull request by exception**
### Description

Implements the `getAvailableSlots` handler exported by the `@calcom/platform-libraries` package.

### CODE REVIEW

File: apps/api/v2/src/modules/slots/inputs/get-available-slots.input.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  @IsBoolean()
+  @IsOptional()
+  debug?: boolean;
+
+  @IsNumber()
+  @IsOptional()
+  duration?: number;
+
+  @IsBoolean()
+  isTeamEvent!: boolean;
```
Comment (keithwillcode) [inline apps/api/v2/src/modules/slots/inputs/get-available-slots.input.ts:35]: Curious why the client would have to specify if it’s a team event or not?

Comment (keithwillcode) [reply apps/api/v2/src/modules/slots/inputs/get-available-slots.input.ts:35]: It feels weird that the person calling it needs to know that versus just passing the eventTypeId and we figure it out

Comment (exception) [reply apps/api/v2/src/modules/slots/inputs/get-available-slots.input.ts:35]: That makes sense, I'll change logic for that.

File: apps/api/v2/src/modules/slots/services/slots.service.ts [hunk 2]
```diff
@@ -40,4 +40,13 @@ export class SlotsService {
 
     return uid;
   }
+
+  async checkIfIsTeamEvent(eventTypeId?: number) {
+    if (!eventTypeId) return false;
+
+    const event = await this.eventTypeRepo.getEventTypeById(eventTypeId);
+    if (event?.teamId) return true;
```
Comment (keithwillcode) [inline apps/api/v2/src/modules/slots/services/slots.service.ts:48]: Nit: can be 1 line of code since you are evaluating a Boolean to return a Boolean

**Review by ThyMinimalDev — APPROVED** [review]

**Pull request merged**